charts/lagoon-core/values.yaml

```diff
@@ -3,6 +3,7 @@
 # elasticsearchHost:
 # harborAdminPassword:
 # harborURL:
+# k8upBackupBucketName:
```

Comment:
Could we make this value optional so that the bucket name can be autogenerated by default? e.g. generate a bucket name from something like `{{ include "lagoon-core.fullname" . }}-{{ randAlphaNum 4 }}` if the value isn't set, but allow it to be overridden if this values is set?